miso-web/src/main/webapp/WEB-INF/pages/editTransfer.jsp

```diff
@@ -18,6 +18,34 @@
   <button id="save" type="button" class="fg-button ui-state-default ui-corner-all">Save</button>
 </h1>
 
+<div class="sectionDivider" onclick="Utils.ui.toggleLeftInfo(jQuery('#note_arrowclick'), 'notediv');">Quick Help
+  <div id="note_arrowclick" class="toggleLeft"></div>
+</div>
+<div id="notediv" class="note" style="display:none;">
+  <p>
+    A transfer shows the change of custody for a set of items. There are three types of transfers,
+    and the type that a transfer is classified as depends on the sender and recipient:
+  </p>
+  <table>
+    <thead>
+      <tr>
+        <th>Type</th><th>Sender</th><th>Recipient</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Receipt</td><td>External Lab</td><td>Internal Group</td>
```

Comment:
This is not a blocking question, merely a curiosity: Is the distinction between "External Lab" here and "External Entity" (in Distribution below) meaningful?

Reply:
Yes, "external lab" (for receipt) must be selected from a defined list, whereas "external entity" is a free text entry and could be a person or lab or raccoon, etc. I think this becomes clearer when you look at the controls below as well.

I'm always open to suggestions for wording to make things clearer though! 😉

Reply:
_adds "set default text in the external entity field to :raccoon: " to my MISO April Fool's wishlist_